### ISSUE TITLE
Reinstall of VS Code is failing

### ISSUE THREAD

**Issue by clsmith70** [Microsoft/vscode#32633]
- VSCode Version:  x64 1.15.0
- OS Version:  Win 10 Ent x64

![image](https://user-images.githubusercontent.com/6304269/29362670-e4fb237c-825a-11e7-8d3e-caffe7f7a14a.png)

Steps to Reproduce:

1.  Uninstall Code
2.  Reinstall Code - fails to finish

Reproduces without extensions: N/A

**Comment by joaomoreno** [comment]
Can you try to completely uninstall VS Code and reinstall it?

**Comment by clsmith70** [comment]
I uninstalled before trying to install.  I was getting javascript errors trying to launch it and could not find any posts about that problem.  I uninstalled VS Code and tried to reinstall and have even rebooted.  I just tried to install again after a fresh start this morning and I still get the same error.  When I check the path the uc.micro folder isn't even there.  It is like the installer can't create that one folder.
